Gemfile

```diff
@@ -30,7 +30,7 @@ gem 'whenever', require: false
 gem 'multi_json'
 gem 'rack-cors', :require => 'rack/cors'
 gem 'net-sftp'
-gem 'vets_json_schema', '1.0.0'
+gem 'vets_json_schema', git: 'https://github.com/department-of-veterans-affairs/vets-json-schema', ref: 'cdc48375d79cce57ceae86c1095e7a7c00c27324'
```

Comment:
From the gem docs, looks like this is legal:

`:git => 'https://github.com/rails/rails.git', :tag => 'v2.3.5'`

Seems easier to understand than the hash ref. 

At an old job we did this (with Python's pip), and we did run into occasional problems where people would think they had made a tag but not actually pushed the tag to Github. But in that case we were bumping tags multiple times per day.